extensions/ql-vscode/src/vscode-tests/factories/remote-queries/shared/skipped-repositories.ts

```diff
@@ -0,0 +1,30 @@
+import { faker } from '@faker-js/faker';
+import {
+  VariantAnalysisSkippedRepositories,
+  VariantAnalysisSkippedRepositoryGroup
+} from '../../../../remote-queries/shared/variant-analysis';
+
+export function createMockSkippedRepos(): VariantAnalysisSkippedRepositories {
+  return {
+    accessMismatchRepos: createMockSkippedRepoGroup(),
+    noCodeqlDbRepos: createMockSkippedRepoGroup(),
+    notFoundRepos: createMockSkippedRepoGroup(),
```

Comment:
Should we ensure that this matches the actual data we receive and make the `notFoundRepos` mock separate/ The `notFoundRepos` does not have an `id` or `private`, while all the other groups will always have both.

Reply:
Ah, I wonder why this wasn't caught by our type system. I've added a separate method to create this: https://github.com/github/vscode-codeql/pull/1559/commits/7dc5eebcc140ae4be86412cb32464cdfcf14c616

Reply:
It seems like we don't have a different type for the not found repos in the `shared` `VariantAnalysis`, while we do have a separate type in the `gh-api` `VariantAnalysis`. It might make sense to create a separate type for it, but that's independent of this PR.